torch/csrc/deploy/CMakeLists.txt

```diff
@@ -46,3 +46,10 @@ target_include_directories(test_deploy_lib BEFORE PRIVATE ${PYTHON_INC_DIR})
 add_executable(deploy_benchmark ${DEPLOY_DIR}/example/benchmark.cpp)
 target_include_directories(deploy_benchmark PRIVATE ${PYTORCH_ROOT}/torch)
 target_link_libraries(deploy_benchmark PUBLIC torch_deploy)
+
```

Comment:
From the CI bots, it looks like this step has to be run after pytorch is installed to the system. Otherwise, `import torch` will fail. Trying to figure out where could be the right phase to run this script.